oak_functions/loader/src/server.rs

```diff
@@ -908,6 +920,16 @@ impl ChannelSwitchboard {
     fn get_mut(&mut self, channel_handle: &ChannelHandle) -> Option<&mut Endpoint> {
         self.0.get_mut(channel_handle)
     }
+
+    // By removing all endpoints from the ChannelSwitchboard their connected endpoints will know
+    // that they will not receive any more messages and get an error when sending messages.
+    fn unregister_all(&mut self) {
```

Comment:
@tiziano88 WDYT?

Reply:
I still think this is unnecessary, but perhaps it is not too bad to keep this method that just clears the switchboard, but again with comments saying that it is not actually necessary, etc.?